test/controllers/AccountHomeControllerSpec.scala

```diff
@@ -109,6 +126,8 @@ class AccountHomeControllerSpec extends SpecBase with MockitoSugar with BeforeAn
 
         status(result) mustEqual SEE_OTHER
       }
+
+      verifyNoInteractions(mockAuditService)
```

Comment:
As this is an entirely different endpoint, I think adding an assertion specifically that this isn't called is a little redundant